### DISCUSSION TITLE
netbox-secrets updated for Netbox v4.1

### DISCUSSION THREAD

**Discussion by vietsoulz** [Onemind-Services-LLC/netbox-secrets#158] [General]
Hello, I recently upgraded Netbox to v4.1.0 and is really intrigued by the netbox-secrets plugin for Netbox.  I tried setting it up but unfortunately the plugin's version 2.0.3 is currently not supported on Netbox v4.1.0 yet.  Is there a roadmap to update the plugin to be supported on 4.1?

**Comment by abhi1693** [comment]
Yes, it's already in the pipeline. We are facing challenges due to the recent graphql changes in netbox which has delayed the release

**Comment by okoelbl** [comment]
Hello!
Since this thread matches the current issue which prevents upgrading from netbox 4.5.4 to 4.5.5 with netbox-docker 4.0.2, I'm writing here.
I saw that there is already a [pull request](https://github.com/Onemind-Services-LLC/netbox-secrets/pull/239) which implements the new GraphQL library.

In general, since we use this plugin about one and a half year now, I recognize that the release cycles of this plugin is often slowing down our upgrade process to next netbox releases. Is there any chance that you can adopt a faster release cycle in the future? Also please let me know if there is anything our team can contribute to.
